lib/inline_svg/webpack_asset_finder.rb

```diff
@@ -38,8 +47,8 @@ def dev_server_asset(file_path)
     end
 
     def fetch_from_dev_server(file_path)
-      http = Net::HTTP.new(Webpacker.dev_server.host, Webpacker.dev_server.port)
-      http.use_ssl = Webpacker.dev_server.https?
+      http = Net::HTTP.new(wrapper.dev_server.host, wrapper.dev_server.port)
+      http.use_ssl = wrapper.dev_server.try(:https?) # `dev_server.https?` is not available on Shakapacker >= 8
```

Comment:
We also would like to see Shakapacker 8 support.

`Shakapacker::DevServer` has a `protocol` method which can be `http` or `https`. This could be used for Shakapacker 8 to set `use_ssl`

Reply:
I would propose if it is possible to re-introduce the `https?` predicate on Shakapacker

Reply:
Let's wait for feedback here: shakacode/shakapacker#486

Reply:
I agree that would be the nicest solution, but since `8.0.0` is already released that won't work for that particular version. In our own codebase we actually monkey patch that back, which `inline_svg` would also have to do.

Reply:
Switched to `protocol` for both webpacker and shakapacker, since it is available from v3